### PR TITLE
HTML Streaming

### PR DESCRIPTION
**Pull request by niccolox**
new docs on Bigcommerce integration: HTML Streaming, location optimization, caching and TTFB

### CODE REVIEW

**Review by calvinbrewer — APPROVED** [review]

**Comment by niccolox** [comment]
TTFB FTW: new docs on Bigcommerce integration: HTML Streaming, location optimization, caching and TTFB

File: content/integrations/how-tos/bigcommerce/_index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+aliases:
+  - /how-to/bigcommerce/
+  - /bigcommerce/
+---
+
+Bigcommerce is a popular ecommerce SaaS that benefits from Section performance optmization.
+
+Section delivers speed improvements to Bigcommerce through location optimization, HTML Streaming and caching of static assets using Varnish Cache.
+
+Section HTML Streaming for Bigcommerce provides dramatic improvement of TTFB.
```
Comment (mattnthat) [inline content/integrations/how-tos/bigcommerce/_index.md:15]: Improvement is to "TTFB and the entire users page load"?

File: content/integrations/how-tos/bigcommerce/optimize-bigcommerce copy.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+         req.url ~ "\/men\/new-arrivals\/.*" ||
+         req.url ~ "\/outlet\/boys\/.*")){
+
+```
+
+## Location Optimization
+
+Section can locate cached and HTML streamed content in the region closest to the customers end users. 
+
+Such locations include:
```
Comment (mattnthat) [inline content/integrations/how-tos/bigcommerce/optimize-bigcommerce copy.md:44]: We should probably just let AEE handle any location decisions? No need to mention locations as customers will run in the correct location for their workload automatically?

File: content/integrations/how-tos/bigcommerce/ttfb-bigcommerce.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Time To First Byte is an important metric for user experience as people do not wait for slow pages to load. 
+
+1. TTFB is a metric of the time taken for a user to receive the first byte.
+2. Start render time is crucial because it represents the time the user is waiting while looking at a blank screen.
+3. There is a high correlation between start render time and bounce rate, so significantly improving this metric has tangible business benefits.
+
+## HTML Streaming
+
+HTML Streaming uses Varnish ESI and OpenResty to punch a hole in the HTML DOM and can stream the HTML behind the title tag, but allows the loading and caching of the head of the document.
```
Comment (mattnthat) [inline content/integrations/how-tos/bigcommerce/ttfb-bigcommerce.md:21]: Maybe no need to mention the underlying technologies? Customer doesnt have to worry about these things..

**Pull request merged**